addons/rose/addon/modifiers/code-mirror.js

```diff
@@ -9,6 +9,8 @@ import 'codemirror/addon/edit/matchbrackets';
 import 'codemirror/addon/edit/closebrackets';
 import 'codemirror/addon/selection/active-line';
 import 'codemirror/addon/display/autorefresh';
+import 'codemirror/addon/lint/lint'
```

Comment:
Asking for curisotiy: what are these linters for? for the editor itself?

Reply:
Yes. I saw that CodeMirror allows for a live linter within the editor itself. I had to add this base lint import to also use the jsonlint that we are using for this component.